algorithms/sort/bogo_sort.py

```diff
@@ -0,0 +1,22 @@
+import random
+
+def bogo_sort(arr):
+    '''Bogo Sort
+        Best Case Complexity: O(n)
+        Worst Case Complexity: O(∞)
+        Average Case Complexity: O(n-n!)
+    '''
+    while not inorder(arr):
+        print(arr)
+        random.shuffle(arr)
+    return arr
+
+def inorder(arr):
```

Comment:
please rename it `is_sorted()` and define it inside the `bogo_sort` as its inner function (it shouldn't accessible outside the function).